src/ScrumBoardItBundle/Security/GitHubAuthenticator.php

```diff
@@ -0,0 +1,45 @@
+<?php
+namespace ScrumBoardItBundle\Security;
+
+use ScrumBoardItBundle\Security\AbstractTokenAuthenticator;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class GitHubAuthenticator extends AbstractTokenAuthenticator
+{
+
+    public function checkCredentials($credentials, UserInterface $user)
+    {
+        $login = $user->getUsername();
+        $password = $credentials['password'];
+        $user->setHash("$login:$password");
+        $url = $this->data['host'] . 'user';
+        
+        $ch = curl_init();
+        curl_setopt($ch, CURLOPT_URL, $url);
+        curl_setopt($ch, CURLOPT_HTTPHEADER, [
+            'Authorization: Basic ' . $user->getHash() .
+            "\naccept: application/json
+            \naccept-language: en-US,en;q=0.8
+            \nuser-agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/49.0.2623.108 Chrome/49.0.2623.108 Safari/537.36"
```

Comment:
Please split the headers with commas instead of carriage returns.
In addition, only "Authorization: Basic [...]" and "Accept: application/vnd.github.v3+json" are usefull